dod-ccpo/atat-web-ui
# PR 1884 - feature/AT-9184-fair-opportunity-summary-page-updates

**To TEST**
Start a new package and ensure the validation work on the following three items..
- [x] selected CSP
- [x] remove barrier radio buttons
- [x] POCs that are `new`

AT-9184
